### ISSUE TITLE
Alternatives to Parcel-Bundler 1.12.5?

### ISSUE THREAD

**Issue by aacraf** [unborn-andy/uib-template-vue-navbar#2]
Hello, thank you for this template. It's a very good start point to use UIBuilder.

I am having some problems related to source maps with some modules and I found that parcel-bundler can't resolve them.

Upgrading to Parcel 2 won't work since it's more to Vue3 and webpack requires some configuration...

I was wondering if you can think about an alternative bundler to use this template...

**Comment by unborn-andy** [comment]
unfortunatly i havent used anything else except Parcel. I know there is Snowpack, webpack, Vue CLI and some others) 
I think i did some tests for a [Vue 2 transformer plugin for Parcel 2 ](https://www.npmjs.com/package/parcel-transformer-vue2) but it was a while back and i dont remember what the issue was and i reverted back to parcel-bundler (Parcel 1)

So the base navbar template that i shared builds without errors .. its just when you use some other modules that you have problems with the source maps ?

**Comment by aacraf** [comment]
Yes, the default template works perfectly. When I add some modules it crashes. However, I found that is a bug and adding --no-source-maps to the parcel script can be solved.

I also tried to use Vite using this template https://github.com/newonejoe/node-red-uibuillder-vite-template/tree/main/uibuilder/vue-app and then I added some files from this template. It seems to work for now.

**Comment by unborn-andy** [comment]
Thanks for sharing your app configuration using Vite .. very useful.
I remember seeing your github before when you shared your code on the Node-red forum
https://discourse.nodered.org/t/using-uibuilder-with-vite/57348
Excellent work

**Comment by unborn-andy** [comment]
I did some tests with Vite and i updated the template to use that bundler instead.
Also updated to use `uibuilder.esm.min.js` introduced in uibuilder version 5

`import {Uib, uibuilder} from './../../../node_modules/node-red-contrib-uibuilder/front-end/uibuilder.esm.min.js'`
